content/en/database_monitoring/agent_integration_overhead.md

```diff
@@ -0,0 +1,73 @@
+---
+title: DBM Agent Integration Overhead
+kind: documentation
+description: Learn about the overhead of integrating the Datadog Agent with your database
+
+---
+
+## Overview
+
+Database Monitoring runs as an integration on top of the base Datadog Agent. The default Agent configuration for Database Monitoring is conservative, but you can adjust settings such as the collection interval and query sampling rate to better suit your needs. For most workloads, the Agent represents less than one percent of query execution time on the database and less than one percent of CPU.
```

Comment:
Propose that we change this to:

`Database Monitoring runs on top of the base Datadog Agent. By default, it's configured to perform optimally while ensuring minimal impact on your system. However, you have the flexibility to adjust settings like data collection frequency and query sampling to better suit your needs.`

`For most workloads, the Agent's resource usage remains minimal, accounting for less than one percent of query execution time and CPU usage on your database.`

Reply:
^ cc @jhgilbert

Reply:
Sorry @lu-zhengda, I actually think I should get our legal contact to give this language a quick review (since we're making claims about Agent impact in it).

Reply:
Will do that and follow up here with what he suggests.

Reply:
sounds good